app/views/dashboards/_actions.html.erb

```diff
@@ -51,7 +51,7 @@
     <% end %>
   <% end %>
 
-  <% if policy(:video).new? %>
+  <% if policy(:video).new? && SiteConfig.enable_video_upload %>
```

Comment:
what if we move this check in the policy or wrap them both in a helper?

Reply:
✅